Libplanet.Net.Tests/Protocols/TestTransport.cs

```diff
@@ -244,30 +244,30 @@ async Task DoAddPeersAsync()
                         }
                     }
 
-                    _logger.Verbose("Trying to ping all {PeersNumber} peers.", tasks.Count);
+                    _logger.Verbose("Trying to ping all {PeersNumber} peers", tasks.Count);
                     await Task.WhenAll(tasks);
-                    _logger.Verbose("Update complete.");
+                    _logger.Verbose("Update complete");
                 }
                 catch (DifferentAppProtocolVersionException)
                 {
-                    _logger.Debug("Different version encountered during AddPeersAsync().");
+                    _logger.Debug("Different version encountered during AddPeersAsync()");
                 }
                 catch (PingTimeoutException)
                 {
                     var msg =
-                        $"Timeout occurred during {nameof(AddPeersAsync)}() after {timeout}.";
+                        $"Timeout occurred during {nameof(AddPeersAsync)}() after {timeout}";
```

Comment:
Do we not replace a template string instead of a plain string this time?

Reply:
Eh, it's because this is used both for logging and an exception. Also this being part of the test code, I didn't think too much about it. 🙄